### ISSUE TITLE
Support PEP-484 style to mark incomplete stubs

### ISSUE THREAD

**Issue by rpgoldman** [facebook/pyre-check#104]
I have created a tiny test case as follows:
```
pyre --search-path /Users/rpg/projects/xplan/yeast-gates-data/stubs/ check
 ƛ Found 1 type error!
foo.py:6:25 Undefined attribute [16]: Module `numpy` has no attribute `log10`.
```
I have been using the (fragmentary) numpy stubs done by that groups developers.  At the bottom of the `numpy/__init__.pyi` file in that `stubs` directory, there is this line, which I believe should keep the above error from being emitted:
```
# TODO(shoyer): remove when the full numpy namespace is defined
def __getattr__(name: str) -> Any: ...
```
So either I am doing something wrong and `pyre` is not finding my stubs, or this trick (which was suggested to me by one of the `mypy` developers) does not work with pyre.  Unfortunately, I don't know how to tell the difference (although I am pretty sure I would have gotten a different error message if pyre hadn't found my stubs) or what to do.  Suggestions? Thanks!
[foo.py.txt](https://github.com/facebook/pyre-check/files/2372838/foo.py.txt)
[numpy-init.pyi.txt](https://github.com/facebook/pyre-check/files/2372840/numpy-init.pyi.txt)

**Comment by dark** [comment]
This is indeed a current limitation of Pyre. We do not support the PEP-484 stub annotation used by numpy yet.

**Comment by dark** [comment]
We have just released `v0.0.15` with this feature.
